frontend/packages/pipelines-plugin/integration-tests/features/pipelines/create-from-builder-page.feature

```diff
@@ -194,9 +195,10 @@ Feature: Create the pipeline from builder page
         Scenario: Create pipeline with optional Workspaces: P-02-TC13
             Given user is at Pipeline Builder page
              When user enters pipeline name as "pipe-opt-workspace"
-              And user clicks on Add task
+              And user clicks Add task button under Tasks section
               And user searches "git-clone" in quick search bar
-              And user clicks on Add in "git-clone" task
+              And user selects "git-clone" from git community
+              And user clicks on Install and add button
```

Comment:
Just to clarify, `git-clone` task is used in multiple scenario, so if git-clone task is Installed and added from the previous scenario, then here you might see only `Add` button not `Install and add` as it is already installed in the cluster.